package.json

```diff
@@ -98,23 +98,24 @@
 		"gatsby-transformer-remark": "^2.3.12",
 		"gatsby-transformer-sharp": "^2.1.19",
 		"gfm-code-blocks": "^1.0.0",
+		"globby": "^9.2.0",
 		"jsonld": "^1.6.0",
 		"make-dir": "^2.1.0",
+		"marked": "^0.6.2",
 		"ncp": "^2.0.0",
 		"node-sass": "^4.12.0",
 		"normalize.css": "^8.0.1",
 		"object-hash": "^1.3.1",
 		"prismjs": "^1.16.0",
 		"prop-types": "^15.7.2",
 		"react": "^16.8.6",
+		"react-accessible-accordion": "^3.0.0",
```

Comment:
Note to self: 
- replace this with a summary details component, do not next link into button.